src/ssd1306.erl

```diff
@@ -84,6 +84,19 @@ stop(SSD1306) ->
 clear(SSD1306) ->
     gen_server:call(SSD1306, clear).
 
+%%-----------------------------------------------------------------------------
+%% @param       SSD1306 a reference to the SSD1306 instance created via start
+%% @returns     ok | {error, Reason}
```

Comment:
You are missing the Mode parameter here.

Reply:
Added that, hopefully that's the right format.

Reply:
Yes. That is the right way to do it. :-)